app/src/main/java/com/bnyro/recorder/ui/Destinations.kt

```diff
@@ -0,0 +1,17 @@
+package com.bnyro.recorder.ui
+
+interface Destination {
```

Comment:
This should be a `sealed class` instead.

Reply:
That seems a good idea.
Using objects with names like Settings, Home feels wierd.